test/testdata/cosigned/valid/valid-policy.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: cosigned.sigstore.dev/v1alpha1
+kind: ClusterImagePolicy
+metadata:
+  name: image-policy
+spec:
+  images:
+  - regex: images.*
+    authorities:
+    - key:
+        data: "---another-public-key---"
+  - glob: image**
+    authorities:
+    - keyless:
+        ca-key:
+          secretRef:
+            name: ca-key-secret
+            namespace: some-namespacemak
+    - keyless:
+        identities:
+        - issuer: "issue-details"
+          subject: "subject-details"
+    - keyless:
+        identities:
+        - issuer: "issue-details1"
+    - key:
+        data: "---some-key---"
```

Comment:
I created this:
https://github.com/sigstore/cosign/issues/1581
to track tighter validations.
But let's get this in, it's a great starting point!